docs/development/contributing.rst

```diff
@@ -87,7 +87,7 @@ after this with ``git push -f``.
 Commit Message formatting
 -------------------------
 To ensure that all commit messages in the master branch follow a specific format, we
-enforce that all commit messages must follow the following format:
+request that your first commit follow the following format:
```

Comment:
We should add here that PR title _must_ contain that format, as right now we're only mentioning that below and IMHO it's not entirely obvious that "title follows the format we suggest" talks about commit title format.

Also I wonder if we should add some blurb on re-running PR title check to PR template just in case.

Reply:
Did both.